modules/nw-multus-remove-pod.adoc

```diff
@@ -3,86 +3,26 @@
 // * networking/multiple_networks/removing-pod.adoc
 
 [id="nw-multus-remove-pod_{context}"]
-= Removing a Pod from an additional network
+= Removing a pod from an additional network
 
-You can remove a Pod from an additional network.
+A pod can be removed from an additional network only by deleting the pod.
```

Comment:
Is there a reason you changed this from active to passive voice? Usually we try to go the other direction.

Would this have the same meaning?

```
You can only remove a pod from an additional network by deleting the pod.
```